src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Date.February": "Février",
+  "FSC.Date.March": "Mars",
+  "FSC.Date.April": "Avril",
+  "FSC.Date.May": "Mai",
+  "FSC.Date.June": "Juin",
+  "FSC.Date.July": "Juillet",
+  "FSC.Date.August": "Août",
+  "FSC.Date.September": "Septembre",
+  "FSC.Date.October": "Octobre",
+  "FSC.Date.November": "Novembre",
+  "FSC.Date.December": "Décembre",
+  "FSC.Time.Current": "Heure actuelle",
+  "FSC.Moon.Phase.New": "Nouvelle Lune",
+  "FSC.Moon.Phase.WaxingCrescent": "Croissante",
+  "FSC.Moon.Phase.FirstQuarter": "Premier trimestre",
+  "FSC.Moon.Phase.WaxingGibbous": "Gibbeuse Croissante",
+  "FSC.Moon.Phase.Full": "Pleine Lune",
+  "FSC.Moon.Phase.WaningGibbous": "Gibbeuse Décroissante",
+  "FSC.Moon.Phase.LastQuarter": "Dernier trimestre",
+  "FSC.Moon.Phase.WaningCrescent": "Décroissante",
+  "FSC.Configuration.Title": "Configuration du calendrier",
+  "FSC.Configuration.Description": "Ce formulaire vous permet de personnaliser votre calendrier de jeux en précisant l'année en cours, le nombre de mois, le nom de chaque mois et le nombre de jours de chaque mois.",
+  "FSC.Configuration.Help": "Pour obtenir de l'aide sur la configuration du calendrier, assurez-vous de consulter la <a href='https://github.com/vigoren/foundryvtt-simple-calendar/tree/main/docs/Configuration.md ' target='_blank'>documentation!</a>",
+  "FSC.Configuration.Save": "Enregistrer la configuration",
+  "FSC.Configuration.General.Title": "Paramètres généraux",
+  "FSC.Configuration.General.PreDefined": "Calendriers prédéfinis",
+  "FSC.Configuration.General.PreDefinedHelp": "Choisissez l'un des calendriers prédéfinis pour démarrer la configuration plus rapidement.",
+  "FSC.Configuration.General.GameWorldTime": "Intégration du temps dans le monde du jeu",
+  "FSC.Configuration.General.GameWorldTimeHelp": "Dites à Simple Calendar comment s'intégrer au temps du monde de jeu et répondre aux mises à jour du temps du monde de jeu.",
+  "FSC.Configuration.General.None": "Aucun : N'est pas lié au temps du monde de jeu.",
+  "FSC.Configuration.General.Self": "Self: Contrôlez le temps du monde de jeu en ignorant les mises à jour des autres modules.",
+  "FSC.Configuration.General.ThirdParty": "Module tiers : Suit uniquement les mises à jour du temps du monde de jeu provenant d'autres modules.",
+  "FSC.Configuration.General.Mixed": "Hybride : Mettre à jour le temps du monde de jeu et suivre les mises à jour effectuées à partir d'autres modules.",
+  "FSC.Configuration.General.PF2ESync": "Pathfinder 2E : synchronisation du temps du monde de jeu",
+  "FSC.Configuration.General.PF2ESyncHelp": "Cela forcera Simple Calendar à rester synchronisé avec le temps du monde de jeu intégrée de Pathfinder 2E.",
+  "FSC.Configuration.General.ShowClock": "Afficher l'horloge",
+  "FSC.Configuration.General.ShowClockHelp": "Si pour afficher l'horloge sous le calendrier, si coché, les commandes pour changer les heures, les minutes et les secondes s'afficheront également.",
```

Comment:
I believe there is an error in the original string. It should probably be translated to "Pour afficher l'horloge sous le calendrier, si coché. Les commandes pour changer les heures, les minutes et les secondes s'afficheront également." (Show the time clock below the calendar, if checked. The controls for changing hours, minutes and seconds will also be shown.)